blocks/v2-footer/v2-footer.js

```diff
@@ -0,0 +1,251 @@
+    return;
+  }
+
+  clearTimeout(timeOut);
+}
+
+function addScrollToTopButton(mainEl) {
+  const scrollToTopButton = createElement('button', ['v2-scroll-to-top'], { title: 'Go to the top of the page' });
+  const svgIcon = document.createRange().createContextualFragment(`
+  <svg xmlns="http://www.w3.org/2000/svg"><use href="#icons-sprite-arrow-right"></use></svg>`);
+  scrollToTopButton.append(...svgIcon.children);
+
+  scrollToTopButton.addEventListener('click', goToTopFunction);
+  window.addEventListener('scroll', () => displayScrollToTop(scrollToTopButton));
+  mainEl.append(scrollToTopButton);
+}
+
+function findList(ele) {
+  if (ele.classList.contains(blockNameTruckList)) {
+    return ele;
+  }
+  return findList(ele.parentElement);
+}
+
+function toggleExpand(targetH3) {
+  const clickedColumn = findList(targetH3);
+  const isExpanded = clickedColumn.classList.contains('expand');
+  const wrapper = targetH3.closest(`.${blockNameTruckList}`);
+  const content = wrapper.querySelector(`.${blockNameTruckList}__items`);
+  if (wrapper === clickedColumn && !isExpanded) {
+    wrapper.classList.add('expand');
+    content.style.maxHeight = `${content.scrollHeight}px`;
+  } else {
+    wrapper.classList.remove('expand');
+    content.style.maxHeight = null;
+  }
+}
+
+export default async function decorate(block) {
+  const cfg = readBlockConfig(block);
+  block.textContent = '';
+
+  const footerPath = cfg.footer || '/footer';
+  const resp = await fetch(`${footerPath}.plain.html`);
+  const html = await resp.text();
+
+  block.innerHTML = html;
+
+  // Eloqua form necessary variables
+  let observer = null;
+  let submitButtonFixed = false;
+  let checkboxFixed = false;
+
+  const footerItems = block.querySelectorAll(`:scope > div .${blockName} > div`);
+
+  const prefooter = footerItems?.[0].querySelector(':scope > div');
+  const truckList = footerItems?.[1].querySelector(':scope > div');
+  const footerMenu = footerItems?.[2];
+  const footerLegal = footerItems?.[3].querySelector(':scope > div');
+
+  const newFooter = createElement('div');
+
+  // Prefooter
+  if (prefooter) {
+    prefooter.classList.add(blockNamePrefooter);
+    newFooter.append(prefooter);
+  }
+
+  // Truck list
+  if (truckList) {
+    truckList.classList.add(`${blockNameTruckList}__wrapper`);
+    truckList.querySelector('ul')?.classList.add(`${blockNameTruckList}__items`);
+    addClassToTitle(truckList, `${blockNameTruckList}__title`);
+    const truckListContent = createElement('div', blockNameTruckList);
+    truckListContent.appendChild(truckList);
+
+    newFooter.append(truckListContent);
+  }
+
+  // Menu: social media + logo + menu list + newsletter form
+  if (footerMenu) {
+    const newMenu = createElement('div', blockNameMenu);
+
+    // Logo
+    // const logo = footerMenu.querySelector('picture');
+    const logo = createElement('div');
+    const logoLink = createElement('a');
+    logoLink.href = 'https://www.macktrucks.com/';
```

Comment:
you can do this `const logoLink = createElement('a', '',  { href:  'https://www.macktrucks.com/'});`

Reply:
Fixed